lib/trusty_cms/extension_loader.rb

```diff
@@ -73,7 +76,8 @@ def load_extension(name)
       begin
         constant = "#{name}_extension".camelize
         extension = constant.constantize
-        extension.unloadable
+        # not sure if this is going to break anything
+        # extension.unloadable
```

Comment:
Evil Line

Reply:
https://stackoverflow.com/questions/34680855/loaderror-unable-to-autoload-constant-applicationcontroller